src/media/inner.rs

```diff
@@ -579,23 +579,23 @@ impl MediaInner {
                 // Find a historic packet that is smaller than this max size. The max size
                 // is a headroom since we can accept slightly larger padding than asked for.
                 let max_size = pad_size * 2;
-                let packet = buffer.historic_packet_smaller_than(max_size)?;
-
-                let seq_no = packet
-                    .seq_no
-                    .expect("this packet to have been sent and therefore have a sequence number");
-
-                // Piggy-back on regular resends.
-                // We should never add padding as long as there are resends.
-                assert!(self.resends.is_empty());
-                self.resends.push_front(Resend {
-                    pt,
-                    ssrc: packet.meta.ssrc,
-                    seq_no,
-                    queued_at: now,
-                });
-
-                return self.poll_packet_resend(now, true);
+                if let Some(packet) = buffer.historic_packet_smaller_than(max_size) {
```

Comment:
Can you avoid reindentation by using let-else and early return?

Reply:
No because the other block we want to run is in the same function

Reply:
The problem before was that this code did an early return here, that's exactly what we don't